internal/cmd/genolm/main.go

```diff
@@ -147,7 +147,7 @@ func run(output, repoRoot, versionStr string) error {
 }
 
 func writeFile(path string, tmpl *template.Template, data interface{}) error {
-	f, err := os.Create(path)
+	f, err := os.Create(path) // #nosec G304
```

Comment:
Can you add a justification to this nosec? Alternatively we could remove the nosec – it looks like this one isn't being flagged with a high enough severity when I run gosec locally.

Reply:
**gosec 2.10.0** was installed on my machine when I originally ran `make lint`.

When I uninstalled **gosec** and re-ran `make lint` the tests resorted to using **gosec 2.9.2** as defined in the Makefile and the lint passed.  I'll amend my PR to remove the `// nosec G304`.